### PR TITLE
Fix crash on NetworkPolicy when matchLabels is missing

### LINKED ISSUE #4497: Viewing any NetworkPolicy results in an error
**Describe the bug**
Since release 5.3.2 I cannot view any NetworkPolicy but get an error

**To Reproduce**
1. Click on any NetworkPolicy
2. See error

**Environment (please complete the following information):**
- Lens Version: 5.3.2.
- OS: Arch Linux
- Installation method: AUR - based on AppImage

** Error stack:**

```
Error stack:

TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at NetworkPolicyDetails.renderIPolicySelector (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:476086:14)
    at http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:476099:22
    at Array.map (<anonymous>)
    at NetworkPolicyDetails.renderNetworkPolicyPeers (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:476097:19)
    at http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:476138:26
    at Array.map (<anonymous>)
    at NetworkPolicyDetails.render (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:476136:24)
    at allowStateChanges (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:2571:12)
    at http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:11008:59
    at trackDerivedFunction (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:3120:18)
    at Reaction.track (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:3649:18)
    at NetworkPolicyDetails.reactiveRender [as render] (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:11006:14)
    at NetworkPolicyDetails.makeComponentReactive (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:11021:25)
    at NetworkPolicyDetails.target.render (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:10934:34)
    at qi (http://e7b3a21a7e92f32280c0dcf53ed54d6c.localhost:33191/build/Lens.js:556376:188)
```

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

fixes #4497

Screen Shot 2021-12-03 at 9 38 36 AM

### CODE REVIEW

**Review by jweak — APPROVED** [review]
Fixes the crash

LGTM

File: src/renderer/components/+network-policies/network-policy-details.tsx [hunk 1]
```diff
@@ -60,20 +62,57 @@ export class NetworkPolicyDetails extends React.Component<Props> {
...
+    return matchExpressions.map(expr => {
+      switch (expr.operator) {
+        case "DoesNotExist":
+        case "Exists":
+          return <li key={expr.key}>{expr.key} ({expr.operator})</li>;
+        case "In":
+        case "NotIn":
+          return (
+            <li key={expr.key}>
+              {expr.key}({expr.operator})
```
Comment (jim-docker) [inline src/renderer/components/+network-policies/network-policy-details.tsx:88]:
```suggestion
              {expr.key} ({expr.operator})
```

**Review by jim-docker — APPROVED** [review]
also now renders details for matchExpressions

File: src/renderer/components/+network-policies/network-policy-details.tsx (hunk 1 above)
Comment (Nokel81) [reply src/renderer/components/+network-policies/network-policy-details.tsx:88]: I'll do that in a follow up PR.

**Pull request merged**